### PR TITLE
Update docs with Array manipulation

### PR DESCRIPTION
**Pull request by ansonkao**
The existing docs are unclear around how to proceed with Array manipulation. I've added an example, and re-organized a couple of existing examples to guide the user in a friendlier sequence.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,7 +117,19 @@ var result = u({ name: { first: 'Susan' } }, person);
 expect(result).to.eql({ name: { first: 'Susan', last: 'West' } });
 ```
 
-#### Multiple updates, including an array
+Arrays elements:
```
Comment (aaronjensen) [inline README.md]: `Array` instead of `Arrays`

**Comment by aaronjensen** [comment]
This is great, thanks! just one small change and then I can merge it.

**Comment by ansonkao** [comment]
Change is done. My pleasure, and thank you for the awesome repo! Cheers :)

**Comment by aaronjensen** [comment]
Thanks, merged.

**Pull request merged**